src/doc/java_client/Examples/TryQuery.rst

```diff
@@ -0,0 +1,32 @@
+.. _TryQuery:
+
+TryQuery
+~~~~~~~~
+
+This example program shows how to use some of VisIt's query capabilities to perform picks and lineouts.
+
+
+Running the program
+"""""""""""""""""""
+
+From a released client to an installed VisIt_ of the same version:
+
+.. code-block:: shell
+
+    java -cp . TryQuery -stay -path /path/to/visit/bin -datapath /path/to/silodata
+
+
+From a devlopment build/java directory:
```

Comment:
```suggestion
From a development build/java directory:
```

Reply:
Oh the joys of cut and paste before spell checking what you are cutting!  Thanks for catching!